charts/refarch-template/charts/frontend/templates/ingress.yaml

```diff
@@ -0,0 +1,60 @@
+{{- if .Values.ingress.enabled -}}
```

Comment:
sollen wir das mit der ingress mit der annotation https://docs.openshift.com/container-platform/4.16/networking/routes/route-configuration.html#nw-ingress-creating-a-route-via-an-ingress_route-configuration oder auch eine openshift route zur Verfügung stellen

Reply:
@hupling wäre schon besser, wenn wir in gitHub ingress verwenden. route ist ja openshift-spezifisch.